form.php

```diff
@@ -38,17 +14,11 @@
   require("nav.php")
   ?>
   contact page
-  <div id="id01" class="modal">
-    <div id = "login_modal">
-      <span onclick="document.getElementById('id01').style.display='none'"class="close" title="Close Modal">&times;</span>
-      <h1>Login</h1>
-      <form action="" method="post" id="login">
-        <p>Username</p><input type="text" name="username" id="box" placeholder="Please enter a username" /><br />
-        <p>Password</p><input type="password" name="password" id="box" placeholder="Please enter a password" /><br />
-        <input type="submit" value="Login"/><br/>
-      </form>  
-    </div>
-  </div>
+  <?php
+  require("login_form.php");
+  require('signup_form.php');
+  ?>
+  <script src="script.js"></script>
```

Comment:
Simplified the contact page by putting the code for the login and signup forms into separate PHP files.